convert2rhel/actions/system_checks/dbus.py

```diff
@@ -34,7 +34,13 @@ def run(self):
         logger.task("Prepare: Check that DBus Daemon is running")
 
         if tool_opts.no_rhsm:
-            logger.info("Skipping the check because we have been asked not to subscribe this system to RHSM.")
+            message = "Skipping the check because we have been asked not to subscribe this system to RHSM."
+            logger.info(message)
+            self.add_message(
+                level="WARNING",
+                id="DBUS_IS_RUNNING_CHECK_SKIP",
```

Comment:
I'm not sure if that `id` makes sense in this context... Sure, we are skipping the check, but it is not because we _know it is running_, it is because the user specified the `no_rhsm` option in the CLI.

I would rename this to 
```suggestion
                id="DBUS_NO_RHSM_CHECK_SKIP",
```

or even 

```suggestion
                id="DBUS_NO_RHSM_SKIP",
```

does that makes sense to you?

Reply:
Yes we can go with this name instead